### PR TITLE
Aerogear 2970

### PR DESCRIPTION
**Pull request by StephenCoady**
- [x] Update authenticate button
- [x] Update background
- [x] Fix alignment of image and button

Screenshot: https://docs.google.com/presentation/d/1FZL-Wy5D_wUo5ADrfhpRBh-Ny2rGauI3_C8SBrlk6rw/edit#slide=id.g3c52d834c7_0_184

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,6 +47,9 @@
     "cordova-plugin-dialogs": "^2.0.1",
     "cordova-plugin-inappbrowser": "^3.0.0",
     "cordova-plugin-ionic-webview": "^1.2.1",
+    "cordova-plugin-iroot": "^0.7.0",
```
Comment (ciaranRoche) [inline package.json]: Do these need to be added back in?

**Comment by wei-lee** [comment]
@ciaranRoche Yes, I think it does look better, and also in the future we may add the notification back in.

File: src/app/app.scss [hunk 2]
```diff
@@ -46,3 +39,16 @@
   position: relative;
   top: 2px;
 }
+
+.authenticate-button {
+  background-color: #3d7185 !important;
```
Comment (wei-lee) [inline src/app/app.scss]: can we create a name color and use it here and everywhere else?

File: src/app/app.scss [hunk 3]
```diff
@@ -16,14 +16,7 @@
...
 .aerogear-background {
-  background-image: -webkit-linear-gradient(top, #2e6980 0%, #4296b4 100%);
-    background-image: -o-linear-gradient(top, #2e6980 0%, #4296b4 100%);
-    background-image: linear-gradient(to bottom, #2e6980 0%, #4296b4 100%);
-    background-repeat: repeat-x;
-    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff2e6980', endColorstr='#ff4296b4', GradientType=0);
-    border-bottom: 1px solid #2e6980;
-    color: #fff;
-    position: relative;
+  background-image: url('../assets/imgs/background-1.png') !important;
```
Comment (wei-lee) [inline src/app/app.scss]: will this scale across different screen size?

**Comment by StephenCoady** [comment]
@wei-lee addressed your comments, added a cover to the background image. can you take another look and if you approve i will merge

**Pull request merged**